### PR TITLE
Enforce consistent terraform formatting

### PR DESCRIPTION
**Pull request by lemmih**
**Summary of changes**
Changes introduced in this pull request:
- Use `terraform fmt` to enforce consistent formatting rules.

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

File: .github/workflows/lintingci.yml [hunk 1]
```diff
@@ -58,3 +58,21 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Setup terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+           path: ~/.tflint.d/plugins
+           key: tflint-${{ hashFiles('.tflint.hcl') }}
```
Comment (LesnyRumcajs) [inline .github/workflows/lintingci.yml:72]: What is this exactly, and why do we need this?

Comment (LesnyRumcajs) [reply .github/workflows/lintingci.yml:72]: More specifically, why do we need to override the defaults, whatever they are?

Comment (lemmih) [reply .github/workflows/lintingci.yml:72]: Oops, that's a copy and paste mistake.

**Review by LesnyRumcajs — APPROVED** [review]

**Review by samuelarogbonlo — APPROVED** [review]
Looks good to me, especially the new "equal sign" indenting for the other files

**Pull request merged**